UnityMirror/UnityServer/UserSettings/Layouts/default-2021.dwlt

```diff
@@ -15,11 +15,11 @@ MonoBehaviour:
   m_PixelRect:
     serializedVersion: 2
     x: 0
-    y: 42.22222
-    width: 1706.6666
-    height: 869.7778
+    y: 42
```

Comment:
Why have many values here changed? Where are they coming from?

Reply:
this is Unity version upgrade